android/src/main/cpp/headers/JNIRegistry.h

```diff
@@ -0,0 +1,40 @@
+//
+// Created by Karol Bisztyga on 2020-03-23
+//
+
+#ifndef REANIMATEDEXAMPLE_JNI_REGISTRY_H
+#define REANIMATEDEXAMPLE_JNI_REGISTRY_H
+
+#include <unordered_map>
+#include <string>
+#include <jni.h>
+#include <tuple>
+
+enum class JNIMethodMode {
+    standard_method,
+    static_method,
+};
+
+class JNIRegistry {
+    JNIEnv* env;
+    // keys are class names
+    std::unordered_map<std::string, jclass> classes;
```

Comment:
I think it's better to have all methods and classes that we use in arrays. It's a really important part of Reanimated.

Reply:
fixed e0fd007